c10/core/InferenceMode.h

```diff
@@ -11,12 +12,16 @@ struct TORCH_API InferenceMode {
   // Note [Expected TLS state in InferenceMode]:
   //   InferenceMode: InplaceOrView not in raw_local_dispatch_key_set.included(),
   //                  Autograd in raw_local_dispatch_key_set.excluded()
+  //                  GradMode is disabled.
   //   NormalMode: InplaceOrView in raw_local_dispatch_key_set.included(),
   //               Autograd not in raw_local_dispatch_key_set.excluded()
+  //               GradMode is enabled by default unless toggled manually through
```

Comment:
nit: there are quite a lot of ways in which the use toggles this (that all route down to GradMode). Not sure how precise you want to be here.